examples/dialog-modal/js/dialog.js

```diff
@@ -212,14 +217,14 @@ aria.Utils = aria.Utils || {};
     this.dialogNode.className = 'hidden';
     this.focusAfterClosed.focus();
 
+    // unwrap the dialog and remove the backdrop
+    this.backdropNode.parentNode.insertBefore(this.dialogNode, this.backdropNode);
```

Comment:
When the dialog closes and using JAWS or NVDA, focus is no longer going back to the focusAfterClose element. Wondering if order of operations is a problem here.